test/gtest/ucs/test_memtrack.cc

```diff
@@ -159,12 +159,15 @@ UCS_TEST_F(test_memtrack, sysv) {
 
 UCS_TEST_F(test_memtrack, memalign_realloc) {
     void* ptr;
+    int ret;
 
-    ptr = ucs_memalign(10, ALLOC_SIZE, ALLOC_NAME);
+    ret = ucs_posix_memalign(&ptr, 10, ALLOC_SIZE, ALLOC_NAME);
+    ASSERT_EQ(ret, 0);
```

Comment:
ASSERT_EQ(0, ret)

Reply:
Actually this gets EINVAL because, according to POSIX, `The value of alignment shall be a power of two multiple of sizeof (void *).`  Does UCX really use non-power-of-2 alignment in practice ?